packages/metro-resolver/src/resolve.js

```diff
@@ -446,7 +446,7 @@ function resolveWindowsPath(modulePath) {
 }
 
 function isRelativeImport(filePath: string) {
-  return /^[.][.]?(?:[/]|$)/.test(filePath);
+  return /^[.][.]?(?:[/]|[\\][\\]|$)/.test(filePath);
```

Comment:
> The original code here is strange (why all the wrapping in [ ]?) and we should fix it.
I thought it strange, too. I was just following the convention established in the original.

> Regardless, is this correct? Wouldn't filePath have just one backslash on Windows? Double backslashes look like a string escaping artifact.
In the linked issue this was the solution.

> I think we should cover this with a Resolver.resolve() test.
I did attempt this, but wasn't sure how to make the tests pass on a Windows system to begin with. On a *nix system the paths don't resolve this way. Ideally I'd just unit test this one function, but can't import it without exporting it from the resolver.js, or using something like Rewire.

Reply:
@motiz88 Do you have any suggestion for the above regarding testing?

Reply:
In the linked issue double backslash is used because it is part of string literal (the first backslash is escape character - i.e string '..\\' consists of 3 characters: two dots and single backslash at the end).  you should use \\  instead of \\\\  (otherwise your expression returns false on relative paths like ..\__generated__\ .  for example this test `/^\.\.?(?:\/|\\\\|$)/.test('..\\__generated__');`  returns false

Reply:
Thanks. I have modified the branch to match against a single backslash.